pkg/api/agentPoolOnlyApi/v20180331/apiloader_test.go

```diff
@@ -5,7 +5,7 @@ package v20180331_test
 
 import (
 	"github.com/Azure/aks-engine/pkg/api"
-	"github.com/Azure/aks-engine/pkg/api/agentPoolOnlyApi/v20180331"
+	v20180331 "github.com/Azure/aks-engine/pkg/api/agentPoolOnlyApi/v20180331"
```

Comment:
The rest of the changes make sense, but I don't understand why this type of change is necessary. The `v20180331` package was already imported and used, so why do we need to pin it this way?

Reply:
We're assuming it's for readability. It makes `goimports` happy, so 🤷‍♂️.